### PR TITLE
changes in build log for nightly details

### PR DESCRIPTION
**Pull request by alokgoswami-ag**
nightly image details for upgrade jobs changes

### CODE REVIEW

File: monitor.py [hunk 1]
```diff
@@ -476,13 +476,13 @@ def get_nightly(build_log_url,build_log_response, job_platform):
         else:
             nightly = job_platform+"-latest-"+ nightly_log_match.group(2)
     else:
-        nightly_initial_log_re = re.compile('(Resolved release {}-initial to (\S+))'.format(job_platform), re.MULTILINE|re.DOTALL)
+        nightly_initial_log_re = re.compile('(Using explicitly provided pull-spec for release {}-initial \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)
```
Comment (Shilpa-Gokul) [inline monitor.py:479]: This changes will work only for release controller triggered upgrade jobs. You should probably add this in a else condition to support both

File: monitor.py [hunk 2]
```diff
@@ -476,13 +476,13 @@ def get_nightly(build_log_url,build_log_response, job_platform):
...
         nightly_initial_log_match = nightly_initial_log_re.search(build_log_response.text)
         if nightly_initial_log_match is None:
             nightly = "Unable to fetch nightly {}-initial information- No match found".format(job_platform)
         else:
             nightly = job_platform+"-initial-"+ nightly_initial_log_match.group(2)
-        nightly_latest_log_re = re.compile('(Resolved release {}-latest to (\S+))'.format(job_platform), re.MULTILINE|re.DOTALL)
+        nightly_latest_log_re = re.compile('(Using explicitly provided pull-spec for release {}-latest \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)
```
Comment (Shilpa-Gokul) [inline monitor.py:485]: For both initial and latest image, check if this matches `re.compile('(Resolved release {}-latest to (\S+))'.format(job_platform), re.MULTILINE|re.DOTALL) `, if it doesnt matches then check `re.compile('(Using explicitly provided pull-spec for release {}-latest \((\S+)\))'.format(job_platform), re.MULTILINE|re.DOTALL)`, if this also doesnt match return Unable to fetch

**Review by Shilpa-Gokul — CHANGES_REQUESTED** [review]

File: monitor.py [hunk 3]
```diff
@@ -476,16 +476,27 @@ def get_nightly(build_log_url,build_log_response, job_platform):
         else:
             nightly = job_platform+"-latest-"+ nightly_log_match.group(2)
     else:
+        nightly=""
```
Comment (KeerthanaAP) [inline monitor.py:479]: Do we need this initialisation here. If not can we remove this.

**Comment by KeerthanaAP** [comment]
@alokgoswami-ag Could you add the output of fetchingly nightly details in the description.
Add both the cases 
1.Release controller triggered upgrade jobs
2. Periodic jobs

**Comment by alokgoswami-ag** [comment]
``` --------------------------------------------------------------------------------------------------
4.11 to 4.12 upgrade
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.12-upgrade-from-nightly-4.11-ocp-ovn-remote-libvirt-ppc64le/1818678223752400896
Nightly info- ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.11.0-0.nightly-ppc64le-2024-07-06-014054 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.12.0-0.nightly-ppc64le-2024-07-20-042241
Lease Quota- Failed to fetch lease information
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
Some steps failed:                           
ERRO[2024-07-31T16:02:59Z] 
  * could not initialize namespace: timed out waiting for image pull secrets 
INFO[2024-07-31T16:02:59Z] Reporting job state 'failed' with reason 'initializing_namespace'


0/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.12 libvirt
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.12-ocp-e2e-ovn-remote-libvirt-ppc64le/1818436838092705792
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.12.0-0.nightly-ppc64le-2024-07-20-042241
Lease Quota- libvirt-ppc64le-1-0
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
Some steps failed:                           
ERRO[2024-07-31T03:08:15Z] 
  * could not run steps: step ocp-e2e-ovn-remote-libvirt-ppc64le failed: ["ocp-e2e-ovn-remote-libvirt-ppc64le" pre steps failed: "ocp-e2e-ovn-remote-libvirt-ppc64le" pod "ocp-e2e-ovn-remote-libvirt-ppc64le-ipi-libvirt-cleanup-pre" failed: pod pending for more than 1h0m0s: containers have not started in 1h0m0.001001324s: test: 


0/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.12 to 4.13 upgrade
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.13-upgrade-from-nightly-4.12-ocp-ovn-remote-libvirt-ppc64le/1818693325989875712
Nightly info- ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.12.0-0.nightly-ppc64le-2024-07-20-042241 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.13.0-0.nightly-ppc64le-2024-07-30-122748
Lease Quota- libvirt-ppc64le-1-2
No crash observed
Build Passed



1/1 deploys succeeded
1/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.12 heavy build
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.12-ocp-heavy-build-ovn-remote-libvirt-ppc64le/1818557420834656256
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.12.0-0.nightly-ppc64le-2024-07-20-042241
Lease Quota- libvirt-ppc64le-0-2
No crash observed
Build Passed



1/1 deploys succeeded
1/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.13 libvirt
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-ovn-remote-libvirt-ppc64le/1818451866166497280
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.13.0-0.nightly-ppc64le-2024-07-30-122748
Lease Quota- libvirt-ppc64le-0-0
All nodes are in Ready state
No crash observed
Failed conformance testcases: 
1. [bz-etcd][invariant] alert/etcdHighCommitDurations should not be at or above info
All symptom_detection testcases passed



1/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.13 to 4.14 upgrade
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.14-upgrade-from-nightly-4.13-ocp-ovn-remote-libvirt-ppc64le/1818708504400105472
Nightly info- ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.13.0-0.nightly-ppc64le-2024-07-30-122748 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.14.0-0.nightly-ppc64le-2024-07-31-152508
Lease Quota- libvirt-ppc64le-1-3
All nodes are in Ready state
No crash observed
Failed conformance testcases: 
1. [sig-scheduling][Early] The HAProxy router pods [apigroup:route.openshift.io] should be scheduled on different nodes [Suite:openshift/conformance/parallel]
All monitor testcases passed
All symptom_detection testcases passed



1/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.13 heavy build
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-heavy-build-ovn-remote-libvirt-ppc64le/1818572497189081088
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.13.0-0.nightly-ppc64le-2024-07-30-122748
Lease Quota- libvirt-ppc64le-1-1
No crash observed
Build Passed



1/1 deploys succeeded
1/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.14 libvirt
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-remote-libvirt-ppc64le/1818467104173592576
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.14.0-0.nightly-ppc64le-2024-07-31-003841
Lease Quota- libvirt-ppc64le-1-1
All nodes are in Ready state
No crash observed
Failed conformance testcases: 
1. [sig-scheduling][Early] The HAProxy router pods [apigroup:route.openshift.io] should be scheduled on different nodes [Suite:openshift/conformance/parallel]
Failed monitor testcases:
1. [Jira:"Network / ovn-kubernetes"] monitor test pod-network-avalibility setup
2. [sig-network] can collect pod-to-service poller pod logs
3. [sig-network] can collect host-to-service poller pod logs
All symptom_detection testcases passed



1/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.14 powervs
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-ovn-ppc64le-powervs/1818587585581158400
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.14.0-0.nightly-ppc64le-2024-07-31-094205
Lease Quota- syd04
Not all worker nodes are up and running
No crash observed
Cluster Creation Failed
level=error msg=Bootstrap failed to complete: timed out waiting for the condition
level=error msg=Failed to wait for bootstrapping to complete. This error usually happens when there is a problem with control plane hosts that prevents the control plane operators from creating the control plane.

0/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.14 heavy build
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-heavy-build-ovn-remote-libvirt-ppc64le/1818587589771268096
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.14.0-0.nightly-ppc64le-2024-07-31-094205
Lease Quota- libvirt-ppc64le-2-1
No crash observed
Build Passed



1/1 deploys succeeded
1/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.15 libvirt
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-remote-libvirt-ppc64le/1818481819062898688
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-020205
Lease Quota- libvirt-ppc64le-1-2
No crash observed
Build Passed



1/1 deploys succeeded
1/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.15 powervs original
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-original/1818617766014357504
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-115653
Lease Quota- Failed to fetch lease information
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
Some steps failed:                           
ERRO[2024-07-31T12:02:49Z] 
  * could not initialize namespace: timed out waiting for image pull secrets 
INFO[2024-07-31T12:02:49Z] Reporting job state 'failed' with reason 'initializing_namespace' 


0/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.15 heavy build
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-heavy-build-ovn-remote-libvirt-ppc64le/1818603529179238400
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-094346
Lease Quota- libvirt-ppc64le-0-0
No crash observed
Build Passed



1/1 deploys succeeded
1/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.15 to 4.16 upgrade
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818795528654163968
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-225348 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-234337
Lease Quota- libvirt-ppc64le-2-1
No crash observed
Build Passed


2 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818763490869907456
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-204650 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-213738
Lease Quota- libvirt-ppc64le-0-2
All nodes are in Ready state
No crash observed
All conformance testcases passed
Failed monitor testcases:
1. [bz-networking] pod IPs should not be used by two pods at the same time
All symptom_detection testcases passed


3 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818719683981873152
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-155110 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-184315
Lease Quota- libvirt-ppc64le-0-0
No crash observed
Build Passed


4 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818659497288142848
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-134837 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-144242
Lease Quota- libvirt-ppc64le-2-2
No crash observed
Build Passed


5 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818628169956069376
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-115653 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-123805
Lease Quota- libvirt-ppc64le-2-0
No crash observed
Build Passed


6 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818616151576416256
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-094346 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-115152
Lease Quota- Failed to fetch lease information
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
{  could not initialize namespace: timed out waiting for image pull secrets}

7 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818607224759521280
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-094346 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-111454
Lease Quota- Failed to fetch lease information
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
{  could not initialize namespace: timed out waiting for image pull secrets}

8 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818573441163333632
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-074442 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-090145
Lease Quota- libvirt-ppc64le-2-3
No crash observed
Build Passed


9 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818512875350659072
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-034903 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-050141
Lease Quota- libvirt-ppc64le-2-0
No crash observed
Build Passed


10 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818485802590539776
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-31-020205 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-031231
Lease Quota- libvirt-ppc64le-1-3
All nodes are in Ready state
No crash observed
Failed conformance testcases: 
1. [sig-ci] [Early] prow job name should match platform type [Suite:openshift/conformance/parallel]
All monitor testcases passed
All symptom_detection testcases passed


11 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-nightly-4.15-ocp-ovn-remote-libvirt-ppc64le/1818457052243038208
Nightly info-  ppc64le-initial-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.15.0-0.nightly-ppc64le-2024-07-30-231412 ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-012003
Lease Quota- libvirt-ppc64le-0-2
No crash observed
Build Passed



9/11 deploys succeeded
7/11 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.16 libvirt
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818795529094565888
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-234337
Lease Quota- libvirt-ppc64le-1-3
No crash observed
Build Passed


2 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818763490735689728
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-213738
Lease Quota- libvirt-ppc64le-1-1
Node details not found
No crash observed
18 testcases have failed, please refer to the job link for more information
symptom_detection test suite error message:
Junit test summary file not found


3 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818719683772157952
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-184315
Lease Quota- libvirt-ppc64le-1-0
No crash observed
Build Passed


4 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818659494805114880
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-144242
Lease Quota- libvirt-ppc64le-1-1
No crash observed
Build Passed


5 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818628169838628864
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-123805
Lease Quota- libvirt-ppc64le-0-1
No crash observed
Build Passed


6 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818616151505113088
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-115152
Lease Quota- Failed to fetch lease information
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
{  could not initialize namespace: timed out waiting for image pull secrets}

7 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818607224658857984
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-111454
Lease Quota- Failed to fetch lease information
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
{  could not initialize namespace: timed out waiting for image pull secrets}

8 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818573441071058944
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-090145
Lease Quota- libvirt-ppc64le-1-3
No crash observed
Build Passed


9 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818512875233218560
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-050141
Lease Quota- libvirt-ppc64le-0-0
No crash observed
Build Passed


10 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818485802544402432
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-031231
Lease Quota- libvirt-ppc64le-2-3
No crash observed
Build Passed


11 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-remote-libvirt-ppc64le/1818457052268204032
Nightly info- registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-012003
Lease Quota- libvirt-ppc64le-0-1
No crash observed
Build Passed



9/11 deploys succeeded
8/11 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.16 powervs
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-ovn-ppc64le-powervs-original/1818617768568688640
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-111454
Lease Quota- Failed to fetch lease information
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
{  could not initialize namespace: timed out waiting for image pull secrets}


0/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.16 heavy build
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-heavy-build-ovn-remote-libvirt-ppc64le/1818617771068493824
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.16.0-0.nightly-ppc64le-2024-07-31-111454
Lease Quota- Failed to fetch lease information
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
{  could not initialize namespace: timed out waiting for image pull secrets}


0/1 deploys succeeded
0/1 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.17 powervs capi
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-ovn-ppc64le-powervs-capi/1818753758474014720
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.17.0-0.nightly-ppc64le-2024-07-31-183657
Lease Quota- lon06-powervs-7-quota-slice-0
No crash observed
Build Passed


2 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-ovn-ppc64le-powervs-capi/1818572503912550400
Nightly info- ppc64le-latest-registry.ci.openshift.org/ocp-ppc64le/release-ppc64le:4.17.0-0.nightly-ppc64le-2024-07-31-070037
Lease Quota- lon06-powervs-7-quota-slice-1
Node details not found
No crash observed
Unable to get cluster status please check prowCI UI
level=error msg=failed to fetch Cluster: failed to generate asset "Cluster": failed to create cluster: control-plane machines were not provisioned within 15m0s: client rate limiter Wait returned an error: context deadline exceeded


1/2 deploys succeeded
1/2 e2e tests succeeded
--------------------------------------------------------------------------------------------------
--------------------------------------------------------------------------------------------------
4.14 SNO
1 Job link: https://prow.ci.openshift.org//view/gs/test-platform-results/logs/periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-sno-power/1818496958080225280
Nightly info- multi-latest-quay.io/openshift-release-dev/ocp-release-nightly@sha256:fa2b5c6d811907147abebf59dd30d72181245674b60f1b9984fa7f6642d396ad
Cluster Creation Failed



0/1 deploys succeeded
0/1 e2e tests succeeded
`--------------------------------------------------------------------------------------------------```

File: monitor.py (hunk 3 above)
Comment (alokgoswami-ag) [reply monitor.py]: done

**Review by Shilpa-Gokul — APPROVED** [review]
/lgtm

**Pull request merged**